### ISSUE TITLE
interactive highlight outside dev bounds

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/gravity-force-lab-basics#294]
**Test device**
MacBook Air (m1 chip)

**Operating System**
11.4

**Browser**
chrome

**Problem description**
https://github.com/phetsims/qa/issues/702

In the iFrame, the interactive highlight for Mass Sphere 2 goes outside of dev bounds. This is not seen with Mass Sphere 1.

**Steps to reproduce**
1. In the iframe version of sim, turn on interactive highlights in preferences
2. Move Mass Sphere 2 all the way to the right

**Visuals**
<img width="759" alt="Screen Shot 2021-08-31 at 10 51 46 AM" src="https://user-images.githubusercontent.com/87318828/131533169-309156cb-1588-460d-87c8-474a2f91f30d.png">

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab: Basics‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-rc.2/phet/gravity-force-lab-basics_all_phet.html
Version: 1.1.0-rc.2 2021-08-30 16:27:03 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Safari/537.36
Language: en-US
Window: 1440x659
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

**Comment by jessegreenberg** [comment]
Thanks @Nancy-Salpepi - also true for the focus highlight:
![image](https://user-images.githubusercontent.com/6396244/132551926-78c98328-3f27-41e1-acb8-f80da30296fd.png)

I am seeing it with mass1 as well, depending on the mass value.

@emily-phet @terracoda I am curious if this is of concern to you or if you want to make any changes?

**Comment by jessegreenberg** [comment]
Discussed with Voicing team:

All agreed that this behavior is OK for now. In the future if we notice excess white space for the highlights we can trim things up but generally it is OK for highlights to extend beyond dev bounds.

We quickly looked into trimming up the extra space that is shown in the above images, but nothing was obvious to change quickly.

Closing this issue.
